fix(holesky): docker compose setup

thedevbirb opened the pull request:
Various fixes to the docker compose setup on Holesky due to my skill issues. Ideally to be merged after #316 or together with it.

thedevbirb committed:
fix(holesky): docker compose setup -- grafana, prometheus

merklefruit reviewed (commented):
why are all the dashboards removed?

thedevbirb commented:
> why are all the dashboards removed?

The other dashboards were taken from the commit boost setup in Helder folder but they don't work with our setup. As such I've removed all of them and replaced `dashboard.json` with `bolt_dashboard.json`

merklefruit commented:
> > why are all the dashboards removed?
> 
> The other dashboards were taken from the commit boost setup in Helder folder but they don't work with our setup. As such I've removed all of them and replaced `dashboard.json` with `bolt_dashboard.json`

Ahh perfect
